pkg/driver/driver.go

```diff
@@ -79,18 +81,78 @@ type Driver interface {
 
 	Info() Info
 
-	// SetConfig sets the configuration for the instance.
-	Configure(inst *store.Instance, sshLocalPort int) *ConfiguredDriver
+	// Configure sets the configuration for the instance.
+	Configure(opts ...ConfigOption) *ConfiguredDriver
```

Comment:
I think we want to keep the instance, since that will always be there.

Configure(inst *store.Instance, opts ...ConfigOption) *ConfiguredDriver